_episodes_rmd/08-plot-ggplot2.Rmd

```diff
@@ -40,53 +40,67 @@ it is the most effective for creating publication quality
 graphics.
 
 ggplot2 is built on the grammar of graphics, the idea that any plot can be
-expressed from the same set of components: a **data** set, a
-**coordinate system**, and a set of **geoms**--the visual representation of data
-points.
-
-The key to understanding ggplot2 is thinking about a figure in layers.
-This idea may be familiar to you if you have used image editing programs like Photoshop, Illustrator, or
-Inkscape.
-
-Let's start off with an example:
+built from the same set of components: a **data set**,
```

Comment:
Change #1: Expand the explanation of how ggplot works to make it easier for novices to understand. Also replaced "coordinate system" with "mapping aesthetics" for the following reasons (from issue #542)
- The word "coordinate" is never used again in the lesson, whereas "mapping" shows up 25 times and "aesthetic" 16 times
- None of the examples actually change the coordinate system. (The closest is adjusting the scale with + scale_x_log10() )
- The base ggplot() command seems to function more like a "layer 0" than a coordinate system. (This may be due to an incorrect mental map on my part, but it behaves like the base layer on which everything else is placed.)
- This would make clearer what aes() is actually doing (mentioned briefly in issue #473)